src/constants/tokens.ts

```diff
@@ -250,13 +250,31 @@ export const MoneyMarketIndex: Token = {
   defaultChain: MAINNET.chainId,
 };
 
+export const ic21Index: Token = {
+  name: 'Index Coop Large Cap Index',
+  symbol: 'ic21',
+  image: '',
+  address: '0x1B5E16C5b20Fb5EE87C61fE9Afe735Cca3B21A65',
+  polygonAddress: undefined,
+  optimismAddress: undefined,
+  decimals: 18,
+  url: 'ic21',
+  coingeckoId: 'large-cap-index-ic21',
+  tokensetsId: '',
+  fees: undefined,
+  isDangerous: false,
```

Comment:
Probably, doesn't matter here but let's keep all tokens `isDangerous: true` (not tradable in the US) @0xonramp